programs/dynamic-bonding-curve/src/constants.rs

```diff
@@ -57,6 +57,12 @@ static_assertions::const_assert_eq!(
 
 pub const MAX_MIGRATION_FEE_PERCENTAGE: u8 = 99;
 
+pub const MIN_LOCKED_LP_PERCENTAGE: u8 = 1; // 1%
```

Comment:
lets set it 10%

Reply:
Fixed
Note: 1% was decided previously based on minimum LP lock percentage of onchain configs